### DISCUSSION TITLE
Обсуждение читов и античита

### DISCUSSION THREAD

**Discussion by German4ik-Slab** [Sokol-Games-GmbH/PixelCombats.PublicApi#251] [General]
![Uploading Screenshot_2024-01-20-23-51-22-790_com.miui.videoplayer.jpg…]()
На скриншоте  мы видим игрока,который использует читы,а именно создает неведомые объекты и исключает игроков
Айди:35861B382C01DB04

**Comment by SobakaBarabaka74** [comment]
Как я понял из разговора с ним, античит пока не планируется новый. Сделали анти-бан на первое время, ждите

**Comment by justqstn** [comment]
> Как я понял из разговора с ним, античит пока не планируется новый. Сделали анти-бан на первое время, ждите 

долбаеб не пиши хуйни

**Comment by justqstn** [comment]
хочешь также?

**Comment by SobakaBarabaka74** [comment]
Всё что тебя не ясно - хуйня? Значит ты не очень далек :)

**Comment by forwork10** [comment]
> Как я понял из разговора с ним, античит пока не планируется новый. Сделали анти-бан на первое время, ждите 

также мак в бане😄

**Comment by SobakaBarabaka74** [comment]
В каком бане? Я основу забросил недавно, я на твинке щас, из за того что телефон умер

**Comment by justqstn** [comment]
> В каком бане? Я основу забросил недавно, я на твинке щас, из за того что телефон умер

ваще ты так написал что я бы уже мог доебаться, но ты просто нищук который ни на что не способен

**Comment by SobakaBarabaka74** [comment]
Немощь не сидит в параше и не строчит хуйню на js, в отличии от тебя? 😁

**Comment by justqstn** [comment]
> Немощь не сидит в параше и не строчит хуйню на js, в отличии от тебя? 😁

какая хуйня на жс? а что я по-твоему делаю?
+чот не видно что ты чето делаешь гений кодинга 300iq

**Comment by SobakaBarabaka74** [comment]
Ой ой ой) кроме режимов на жс5, и софта на луа нечего не смог сделать? ( из моих работ по c++ - это реализация строк, за неделю изучения

**Comment by justqstn** [comment]
мак вы забыли, что я повторил почти весь софт спт, как так :(
ну да еще матан и линал знаешь? нет? иди нахуй

**Comment by SobakaBarabaka74** [comment]
Там софт - нехуй делать, 1 действие научиться делать - весь софт готов

**Comment by justqstn** [comment]
и какое же?)
